cpp/src/gandiva/function_registry_string.cc

```diff
@@ -38,6 +41,13 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
       BINARY_RELATIONAL_SAFE_NULL_IF_NULL_UTF8_FN(starts_with),
       BINARY_RELATIONAL_SAFE_NULL_IF_NULL_UTF8_FN(ends_with),
 
+      UNARY_OCTET_LEN_FN(octet_length),
```

Comment:
Are these additions related to decimal multiplication?

Reply:
no - minor cleanup. I moved them from function_registry_arithmetic.cc.